### PR TITLE
refactor: Updates w.r.t. fluent 251 image update.

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

**Review by seanpearsonuk — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

File: src/ansys/fluent/core/session_solver.pyi [hunk 1]
```diff
@@ -34,22 +21,4 @@ class Solver:
     def read_case(self, file_name: str): ...
     def write_case(self, file_name: str): ...
     @property
-    def file(self) -> file: ...
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.pyi:37]: The property settings should be added

File: src/ansys/fluent/core/session_solver.pyi [hunk 2]
```diff
@@ -34,22 +21,4 @@ class Solver:
...
-    @property
-    def solution(self) -> solution: ...
-    @property
-    def results(self) -> results: ...
-    @property
-    def parametric_studies(self) -> parametric_studies: ...
-    @property
-    def current_parametric_study(self) -> current_parametric_study: ...
-    @property
     def parameters(self) -> parameters: ...
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.pyi:24]: This won't also be exposed at top level, I think.

Comment (prmukherj) [reply src/ansys/fluent/core/session_solver.pyi]: Done, Thank you.

File: src/ansys/fluent/core/session_solver.pyi (hunk 1 above)
Comment (prmukherj) [reply src/ansys/fluent/core/session_solver.pyi:37]: Done, Thank you.

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**